scipy/stats/_distn_infrastructure.py

```diff
@@ -1279,108 +1287,69 @@ class rv_continuous(rv_generic):
     shapes : str, optional
         The shape of the distribution. For example ``"m, n"`` for a
         distribution that takes two integers as the two shape arguments for all
-        its methods.
+        its methods. If not provided, shape parameters will be inferred from
+        the signature of the private methods, ``_pdf`` and ``_cdf`` of the
+        instance.
     extradoc :  str, optional, deprecated
         This string is used as the last part of the docstring returned when a
         subclass has no docstring of its own. Note: `extradoc` exists for
         backwards compatibility, do not use for new subclasses.
-    seed : None or int or np.random.RandomState instance, optional
+    seed : None or int or ``numpy.random.RandomState`` instance, optional
         This parameter defines the RandomState object to use for drawing
         random variates.
         If None (or np.random), the global np.random state is used.
-        If integer, it is used to seed the local RandomState instance
+        If integer, it is used to seed the local RandomState instance.
         Default is None.
 
     Methods
     -------
-    ``rvs(<shape(s)>, loc=0, scale=1, size=1)``
-        random variates
+    rvs
+    pdf
+    logpdf
+    cdf
+    logcdf
+    sf
+    logsf
+    ppf
+    isf
+    moment
+    stats
+    entropy
+    expect
+    median
+    mean
+    std
+    var
+    interval
+    __call__
+    fit
+    fit_loc_scale
+    nnlf
```

Comment:
Notice that I'm adding three fit-related methods here, which were not listed as public methods before.